internal/fusefrontend/file.go

```diff
@@ -60,8 +60,8 @@ type file struct {
 
 // NewFile returns a new go-fuse File instance.
 func NewFile(fd *os.File, fs *FS) (nodefs.File, fuse.Status) {
-	var st syscall.Stat_t
-	err := syscall.Fstat(int(fd.Fd()), &st)
+	var st unix.Stat_t
+	err := unix.Fstat(int(fd.Fd()), &st)
 	if err != nil {
 		tlog.Warn.Printf("NewFile: Fstat on fd %d failed: %v\n", fd.Fd(), err)
 		return nil, fuse.ToStatus(err)
```

Comment:
This may also be a problem - `fuse.ToStatus` compares the error it gets against `syscall` and `os` constants - it even checks the type here:
https://github.com/hanwen/go-fuse/blob/9060525339d688b3e8313f25eee96690d7314a60/fuse/misc.go#L54

	switch t := err.(type) {